src/system/data-attributes-product.md

```diff
@@ -63,19 +63,19 @@ The installation used to export this data has the sample data installed, and has
+|use_config_max_sale_qty|Determines if the default configuration setting for maximum quantity is used, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes)<br/>**0** (or blank) — (No)|
 |is_in_stock|Indicates if the product is in stock.|
 |notify_on_stock_below|Specifies the stock level that triggers an _out of stock_ notification.|
-|use_config_notify_stock_qty|Determines if the default configuration setting is used to trigger stock level notification, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes) The value of the qty attribute is a decimal value.<br/>**0** (or blank) — (No) The value of the qty attribute is a whole number (integer).|
+|use_config_notify_stock_qty|Determines if the default configuration setting is used to trigger stock level notification, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes)<br/>**0** (or blank) — (No)|
 |manage_stock|Determines if inventory control is used to manage the product. Values:<br/>**1** — (Yes) Activates full inventory control to manage stock levels of the product.<br/>**0** (or blank) — (No) The system does not keep track of the number of items that are currently in stock.|
-|use_config_manage_stock|Determines if the default configuration setting for managing stock is used, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes) The value of the qty attribute is a decimal value.<br/>**0** (or blank)—(No) The value of the qty attribute is a whole number (integer).|
-|use_config_qty_increments|Determines if the default configuration setting for quantity increments is used, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes) The value of the qty attribute is a decimal value.<br/>**0** (or blank) — (No) The value of the qty attribute is a whole number (integer).|
-|qty_increments|Determines if the product is sold in quantity increments. Options: Yes / No|
-|use_config_enable_qty_inc|Determines if the default configuration setting to enable quantity increments is used, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes) The value of the qty attribute is a decimal value.<br/>**0** (or blank) — (No) The value of the qty attribute is a whole number (integer).|
+|use_config_manage_stock|Determines if the default configuration setting for managing stock is used, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes)<br/>**0** (or blank) — (No)|
+|use_config_qty_increments|Determines if the default configuration setting for quantity increments is used, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes)<br/>**0** (or blank) — (No)|
+|qty_increments|Establishes the number of products that make up a quantity increment.|
```

Comment:
For the wording on this,  I just copied it from https://experienceleague.adobe.com/docs/commerce-admin/config/catalog/inventory.html?lang=en .